modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/input_move_boxes.js

```diff
@@ -140,25 +142,24 @@ AUI.add(
 				_onSelectChange(event) {
 					const instance = this;
 
-					instance._toggleBtnMove(event);
-					instance._toggleBtnSort(event);
+					instance._toggleBtnMove(event.currentTarget);
+					instance._toggleBtnSort(event.currentTarget);
```

Comment:
Are you sure about changing this from `event.target`([here](https://github.com/liferay-frontend/liferay-portal/pull/2941/files#diff-6abdb4b15c2313fa99ec03d41063a1ac14fe197f3abe42cdf4f3f68c71cd19d9L285)) to `event.currentTarget`?

currentTarget = element to which the event is attached
target = element which caused the event to fire

Reply:
Hi @bryceosterhaus, thank you for your comment. Please correct me if I am wrong here. 

I believe `currentTarget` and `target` are the same element at this point here. The reason is because of the code that called it here:
```
instance._leftBox.after(
'change',
A.bind('_onSelectChange', instance)
);

instance._rightBox.after(
'change',
A.bind('_onSelectChange', instance)
);
```
both `target` and `currentTarget` are referencing either leftBox or rightBox if their values are changed from the event. 

And yes I am sure about it because I've verified them at debugging time.

Again, if this is not making any sense to you, please free to ask us to revert it back to the original code.

Reply:
If they are the same thing, let's keep it the same as the original code, that way we avoid any possible regressions. So just need to change currentTarget back to target for that one instance.

Reply:
Yeah. I will undoubtedly revert it. Is there anything else? I appreciate any help you can provide.